sdk/core/Azure.Core/src/DynamicData/MutableJsonElement.cs

```diff
@@ -919,6 +1033,19 @@ public void Set(byte value)
             Changes.AddChange(_path, value);
         }
 
+        public MutableJsonElement SetProperty(string name, byte value)
```

Comment:
Do all these numeric overloads really make sense in terms of compiled code size? Allocating a `long` or `ulong` (for most cases) on the stack may be fine given all the duplicate code to maintain, load, and execute herein. Just a thought.

Reply:
Yeah, it's a good thought.  We have them for parity with JsonElement (see https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.getuint64?view=net-7.0, e.g.), so I think we need to keep them.  Although we can raise the issue again before we make the type public - it's still internal shared source for now.

Reply:
Sounds good. I was more so pondering out loud.